default.env

```diff
@@ -0,0 +1,25 @@
+# Test database
+DT_SQL_TESTUSER="postgres"
+DT_SQL_TESTPASS="password"
+DT_SQL_TESTHOST="localhost"
+DT_SQL_TESTPORT="5432"
+DT_SQL_TESTDBNAME="test_db"
+
+# Dev database
+DT_SQL_USER="postgres"
+DT_SQL_PASS="<REPLACE_ME>"
+DT_SQL_HOST="db"
+DT_SQL_PORT="5432"
+DT_SQL_DBNAME="dt_dev"
+
+# Secrets for the web servers
+# DT_DEFAULT_USER_PASS must be set to be able to run the test suite.
+DT_BACKEND_URL="http://backend:5000"
+DT_DEFAULT_USER_PASS="<REPLACE_ME>"
+DT_FRONT_SECRET_KEY="<REPLACE_ME>"
+DT_JWT_SECRET_KEY="<REPLACE_ME>"
+
+# dtbase weather key
```

Comment:
This is running into the same issue we talked about before, of keeping a separation with particular ingress/models and the base package.